Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -10,6 +10,22 @@ public static class ArrayOperations
         public static void Sort(int[] array)
         {
             // ToDo: implement.
+            if(array.Length > 1 || array != null)
+            {
+                for (int i = 0; i < array.Length - 1; ++i)
+                {
+                    int mindex = i;
+                    for(int j = i; j < array.Length; ++j)
+                    {
+                        if (array[j] < array[mindex])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)